src/main/resources/story/Walmart/walmart-create-user-account.story

```diff
@@ -0,0 +1,9 @@
+Description: Create user account.
+
+Scenario: Signing up a new user
+Given I am on a page with the URL 'https://www.walmart.com/'
+When I wait until element located `By.xpath(//div[@data-automation-id="headerSignIn"])` appears
```

Comment:
It makes sense to use a wildcard - "*"- instead of "div" in xpathes

Reply:
Fixed, changed to "*"